### ISSUE TITLE
Ability to create landing page experience

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#6773]
As a marketer crafting a landing page experience, I can create my page without having to get the assistance of a developer / add custom and non-supported html to the page.

How to implement this?  Is it a feature?  Is it a new "Landing page" content type?

### Tasks:
- [ ] https://github.com/uiowa/uiowa/issues/8803
- [ ] https://github.com/uiowa/uiowa/issues/2665
- [ ] https://github.com/uiowa/uiowa/issues/8804
- [ ] https://github.com/uiowa/uiowa/issues/6858
- [ ] https://github.com/uiowa/uiowa/issues/9352
- [ ] https://github.com/uiowa/uiowa/issues/9356

## Nice to have:
- [ ] Can we uncheck "Disable client-side validation" for webforms? https://github.com/uiowa/uiowa/issues/5129#issuecomment-1478452628
- [ ] https://github.com/uiowa/uiowa/issues/6836
- [ ] https://github.com/uiowa/uiowa/issues/6837
- [ ] Allow `<br>` in BASIC_HTML text format for blockquotes
- [ ] Size modifier classes in UIDS to allow for larger heading sizes https://github.com/uiowa/uids/issues/721

**Comment by bspeare** [comment]
I've updated this issue with some more information and a task list of work that we might need to implement.

**Comment by bspeare** [comment]
This is why we should turn Disable client-side validation back on for sites:
<details>
Webform in a column:

<img width="541" alt="Screenshot 2023-08-11 at 11 43 05 AM" src="https://github.com/uiowa/uiowa/assets/1036433/bc1ec185-656e-4fab-9a05-c9908f53d9bb">


Webform in a column with errors:

<img width="424" alt="Screenshot 2023-08-11 at 11 42 58 AM" src="https://github.com/uiowa/uiowa/assets/1036433/e823006c-48d8-48c1-a40d-eaa4783d372c">
</details>

**Comment by JKurtzlebenUI** [comment]
Tippie would very much like to make use of this site format as well for a Tippie site dedicated exclusively to landing pages like uiowa.edu/go. 

Example: https://uiowa.edu/go/rmi. 

It will make it easy for us to administer dedicated landing pages for advertising & email promotion, linked to our CRM. Thanks!

**Comment by bspeare** [comment]
# Notes on landing page experience

Landing page feature that has functionality to upload a background image to a section, hide footer, navigation, and search.   

## Hiding elements
**Initial slice:**
-  Blocks: ability to show/hide items based on device 
-  Hiding the footer, main navigation, search

##  Background image with form:
**Initial slice:**
-  Background image upload for a section or ability to place a block within a banner
-  Ability to place form over the top of the background image
-  Different/more flexible image layouts 
- 

## Forms
**Initial slice:**
-  Tighter form styling: closer space between fields

**Future slices:**
-  United States at top of country list easily or a maui-synced country field 
-  Copy forms to other sites 
-  Pass RefCode in hidden field 
   - Is this already available?
-  Adjust spacing by device (like on a form, making it not take up the whole screen vertically)  
-  Lightbox form and/or slide-in form triggered by % scroll, time on page, close behavior 


###  After-content region / pre-footer
**Future slices:**
-  Sticky CTA for mobile 

###  Photoswipe
**Future slices:**
-  Disable pop-up for mobile

###  Need more info
**Future slices:**
-  Area of study field that allows you to start typing 
   - Can a user start typing "Business" and find an exact match a more robust version would be a semantic match.  Future AOS feature for uiowa.edu redesign.

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Complicated

 **Discussion:**
 - Journalism is working on landing page currently, trying to understand wether they will use this or how we previously did these
 - inline css on page has been used to hide things previously as well as tightening up spacing on page
 - Would like to move away from that to something more maintainable
 - Feature split likely appropriate here since we don't know if we want access to this for all sites (at least initially)
 - Is there a reason we wouldn't want to adjust spacing on webforms everywhere vs just landing pages?
   - based around idea of banner image with webform on top (want form to fit and look nice in banner) - May want to understand best practices for landing pages before we address this
   - option on block that provides form
 - new content type could be challenging with config - can't share config so would have duplicate configuration
   - Could have a toggle on node edit for "pages" to designate as a "landing page"
 - Wouldn't want "landing pages" to be indexed, included in sitemap, added to menu system
   - These node level differences should be able to be handled with hooks even if we use "page" content type
- config split that enables custom module
- designate page as "landing page"
- logic to hide elements (menus, search, etc.)
- Medium

**Proposed solution or definition of done:**
